cli_tools/diagnostics/main_windows_test.go

```diff
@@ -94,13 +141,16 @@ func stringArrayIncludesString(stringArray []string, target string) bool {
 func TestGatherEventLogs(t *testing.T) {
 	logFolderCh := make(chan logFolder, 2)
 	errCh := make(chan error)
+	expectedFiles := []string{systemLogPath, applicationTextLogFileName, systemTextLogFileName}
```

Comment:
Testing in this manner is a "change-detector" test, which often isn't very valuable. It's testing the internals of the function, rather than the API / contract.

If these are all the files that **a "user"** should expect gatherEventLogs to return, then these should be defined somewhere (somewhere in the code (e.g. a constant) preferably, or in a comment for gatherEventLogs at least). This makes the set of files part of the API / contract and then it's ok to test for the presence of all of them.

Alternatively, perhaps just choose one "important" file that we expect gatherEventLogs to always return, and test for that. This is the approach that this test took before this PR.

Reply:
Makes sense. Reverted back